src/components/Form/formFunctions.ts

```diff
@@ -134,19 +161,22 @@ export const checkboxGroupToArray = (checkedBoxes: booleanGroup): string[] => {
 
 export const checkErrorOnSubmit = (
```

Comment:
![image](https://github.com/VauxhallFoodbank/vauxhall-foodbank/assets/137791994/1fe0a875-4c4b-438f-8e9d-13c9e7a782d0)

Im getting this error when I tried inserting with database error then submit again with an in complete form but with no db error. Might wanna remove the first section of the error message:)